### ISSUE TITLE
Error while trying to install via stack

### ISSUE THREAD

**Issue by EvertVerboven** [joshuaclayton/unused#104]
While trying to instaal unused via stack in a docker container.
I'm running stack install in a elixir:1.6.2 docker container.
```
Error: While constructing the build plan, the following exceptions were encountered:

In the dependencies for unused-0.9.0.0:
    terminal-progress-bar must match >=0.1.1.1 && <0.1.2, but the stack configuration has no specified version  (latest matching version is 0.1.1.1)
needed since unused is a build target.

Some different approaches to resolving this:

  * Consider trying 'stack solver', which uses the cabal-install solver to attempt to find some working build configuration. This can be convenient when dealing with many complicated constraint
    errors, but results may be unpredictable.

  * Recommended action: try adding the following to your extra-deps in /root/.stack/global-project/stack.yaml:

- terminal-progress-bar-0.1.1.1

Plan construction failed.
```

**Comment by halostatue** [comment]
While trying to install with global `stack install unused`, I get more errors. These appear to be fixed in the local install (`hub clone joshuaclayton/unused && cd unused && stack setup && stack install`).

```
Error: While constructing the build plan, the following exceptions were encountered:

In the dependencies for unused-0.9.0.0:
    megaparsec-7.0.5 from stack configuration does not match >=6.5.0 && <7  (latest matching version
                     is 6.5.0)
    terminal-progress-bar must match >=0.1.1.1 && <0.1.2, but the stack configuration has no
                          specified version  (latest matching version is 0.1.1.1)
needed since unused is a build target.

Some different approaches to resolving this:

  * Recommended action: try adding the following to your extra-deps
    in /Users/austin/.stack/global-project/stack.yaml:

- megaparsec-6.5.0@sha256:a1dfed92863607ea9f7609a0578d486761a052a4d01aaff95b90f25da2564346,6119
- terminal-progress-bar-0.1.1.1@sha256:905387c9b73c61c02668a8e51277bda62a36ab314ef905cb199eab308680d383,2407

Plan construction failed.
```

**Comment by gabrielpedepera** [comment]
Did you find a way to resolve this?

**Comment by miketheman** [comment]
The current workaround I've found is to build the project from scratch - cloning the repo and running `stack setup && stack install`.
I'm no Haskell/Stack expert - but I'm guessing that there's a dependency mismatch in the released 0.9.0.0 and the fix provided in #101 ?

**Comment by joshuaclayton** [comment]
@EvertVerboven @gabrielpedepera @halostatue mind giving this another try soon? It should be addressed now (and updated via 0.10.0.0), just waiting on hackage to pick it up.
